### ISSUE TITLE
Wyoming satellite 

### ISSUE THREAD

**Issue by nighi** [Dragon863/EchoCLI#19]
Have you thought about using a echo dot as a Wyoming satellite for Homeassistant's assist. 
I'm wondering what needs to be done and if this could work anyhow.

**Comment by The-May** [comment]
This would be a total gamechanger if implemented.

**Comment by JustAiden9** [comment]
Yeah if this ever gets implemented I would root all of my echo dot 2s
